### PR TITLE
Ruby collections lesson

### PR DESCRIPTION
**Pull request by Jay-zar**

### CODE REVIEW

File: full_name.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+puts "Hello! What is your first name?"
+first_name = gets.chomp.to_s 
+full_name.push(first_name)
+puts "What's you middle name?"
+middle_name = gets.chomp.to_s 
+full_name.push(middle_name)
+puts "What's your last name?"
+last_name = gets.chomp.to_s 
+full_name.push(last_name)
+puts "Hello #{full_name[0]} #{full_name[1]} #{full_name[2]}!"
```
Comment (itsagift) [inline full_name.rb:13]: Looks good!

**Review by itsagift — COMMENTED** [review]
Great job on #2

File: add_up.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+def add_up (variable)
+    start_num = 0 
+    for num in 1..variable
+        start_num += num
+    end
+    end_num = start_num
+end
+
+puts "Hello! Please enter a whole number greater than one: "
+puts add_up(gets.chomp.to_i)
```
Comment (itsagift) [inline add_up.rb:14]: Works great -- seems like you have to call the function 3 times. I'm actually not clear on whether this means asking for 3 different numbers or repeating the same number three time. I assume the first one.

**Comment by itsagift** [comment]
Great job on #4.

File: add_up.rb (hunk 2 above)
Comment (Jay-zar) [reply add_up.rb:14]: Apologies! I'll fix that. Thank you

**Pull request merged**